src/main/java/duke/main/Ui.java

```diff
@@ -0,0 +1,92 @@
+package duke.main;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    private Scanner sc;
+
+    Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Prints a greeting message when the bot is opened.
+     */
+    public void greetingMessage() {
```

Comment:
Maybe this method could be named like a verb, eg. showGreetingMessage() instead?